cli/foundation-models/system/finetune/image-classification/multilabel-classification/hftransformers-fridgeobjects-multilabel-classification.sh

```diff
@@ -169,23 +169,12 @@ az ml online-endpoint create --name $huggingface_endpoint_name $workspace_info
 }
 
 # Deploy model from registry to endpoint in workspace
-az ml online-deployment create --file ./deploy.yaml $workspace_info --set \
+az ml online-deployment create --file ./deploy.yaml $workspace_info --all-traffic --set \
   endpoint_name=$huggingface_endpoint_name model=azureml:$finetuned_huggingface_model_name:$version \
   instance_type=$deployment_sku || {
     echo "deployment create failed"; exit 1;
 }
 
-# get deployment name and set all traffic to the new deployment
```

Comment:
Why are we taking it out? This was added for fixing the deployment failures.
cc: @MadhuM02

Reply:
can you try any of the cli deployments to see if --all-traffic issue is solved??

Reply:
Finetune MultiLabel is successful. 
[https://ml.azure.com/experiments/id/a00aa4c7-bfc4-40d0-8dce-69395609eef1?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-brahma&tid=72f988bf-86f1-41af-91ab-2d7cd011db47]